pkg/gotohelm/transpiler.go

```diff
@@ -885,6 +889,11 @@ func (t *Transpiler) transpileCallExpr(n *ast.CallExpr) Node {
 					Arguments: append([]Node{t.transpileExpr(n.Fun.(*ast.SelectorExpr).X)}, args...),
 				}
 			}
+		} else if typeName, ok := r.Type().(*types.Named); ok {
```

Comment:
I should probably just make a helper for this but I've been writing this type of check as:

```go
typ := r.Type()

// "unwrap" the pointer type.
switch typ.(type) {
case *types.Pointer:
typ = typ.Elem()
}

if _, ok := typ.(TypeYouActuallyWant); ok {
// ...
}
```